models/engine/file_storage.py

```diff
@@ -68,3 +68,21 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """Return an object with a given class and ID"""
+        ret = None
+        objects = self.all(cls)
+        lent = len(cls.__name__) + 1
+        for i in objects:
+            if i[lent:] == id:
+                ret = objects[i]
+        return (ret)
+
+    def count(self, cls=None):
```

Comment:
I think it is a good idea to take into account the case that the class is not valid.
Example: `count(Country)`

Reply:
Thats a good example of a border case that our project does not handle. Thanks for the request! @coding-max

Reply:
It is always a good option to take into account the edge cases, remember that you have to do unitest for this project and using unexpected arguments should always be taken into account :)